src/toolbar.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { startHighlight } from './highlight';
+
+
+const toolbarStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  display: 'flex',
+  flexDirection: 'column',
+  position: 'fixed',
+  top: '100px',
+  right: '0px',
+  padding: '1px'
+};
+
+const toolbarContentStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  display: 'flex',
+  backgroundColor: '#d3d3d3',
+  flexDirection: 'column',
+  right: '0px',
+  border: '1px solid #757575'
+};
+
+const toolbarToggleStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  backgroundColor: '#d3d3d3',
+  width: '30px',
+  right: '0px',
+  padding: '5px'
+};
+
+const toolbarIconStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  opacity: '0.5',
+  width: '45px',
+  height: '45px'
+};
+
+class Toolbar extends Component {
+
+  state = {
+    isActive: false,
+  };
+
+  handleToggle = () => {
+    this.setState(
+      {
+        isActive: !this.state.isActive
+      });
+  };
+
+  render() {
+
+    return (
+      <div style={toolbarStyle} className="toolbar">
+        {!this.state.isActive && <button onClick={this.handleToggle}><img alt="edit" style={toolbarToggleStyle} src="edit.png" className="toolbarToggle" /></button>}
```

Comment:
I tend to prefer ternary operator over &&, which would put this line something like:
{isActive ? null : <button onClick={handleToggle}><img alt="edit" ...></button>}

However, since all the contents of the parent <div> that's being returned are keying off of isActive, it may be better to just use an if statement with 2 different returns:
if (isActive) {
    return ...;
} else {
    return ...;
}

I'd like input from the rest of the group on this though.